lib/pagem.rb

```diff
@@ -98,3 +98,50 @@ def onkeypress_script
     "if(event.keyCode == 13) Pager.setPage(this, '#{@page_variable}', this.value, #{@is_remote});"
   end
 end
+
+# PagemMultiscope class inherits from Pagem and allows pagination of 2 scopes, returning either the paginated
+# set of the first scope with limit and offset, the paginated set of the second scope with limit and offset,
+# or results of a combined scope proc combining both scopes if the page is rendering a combination
+# of both table records on the same page
+# TODO Move this back to the Pagem class and extend it to allow more than 2 scopes
+class PagemMultiscope < Pagem
+  def initialize(first_scope, second_scope, params, opt = {})
```

Comment:
Since most of these params are unused, a better pattern would be to not implement initialize and instead add a hook method in super's initialize like "post_initialize" which is implemented here